FreestarAds-Ogury.template.podspec

```diff
@@ -10,6 +10,8 @@ Pod::Spec.new do |spec|
   spec.vendored_frameworks  = 'build/FreestarAds-Ogury.xcframework'
   spec.dependency "FreestarAds#{prerelease_tag}", "~> 5.12#{beta_tag}"
```

Comment:
@carlosalcala-freestar, is this correct?  is core version 5.12 compatible with ogury adapter?

Reply:
Fixed